src/vaults/legacy/LegacyVaultAdapter.sol

```diff
@@ -94,12 +94,12 @@ contract LegacyVaultAdapter is AsyncVault, ILegacyVaultAdapter, IInvestmentManag
     // --- IInvestmentManager - View functions ---
     /// @inheritdoc IInvestmentManager
     function convertToShares(address, /* vault */ uint256 _assets) public view returns (uint256 shares) {
-        shares = manager.convertToShares(this, _assets);
+        shares = asyncManager().convertToShares(this, _assets);
```

Comment:
While `manager` is correct as well for fully asynchronous vaults for which `asyncManager == manager` holds true, all other calls use `asyncManager()`.